### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by octonawish-akcodes** [MDAnalysis/mdanalysis#4429] [Introductions]
Hi Everyone, I am Abhishek Kumar from India. I am new to this community and looking forward to contribute here and learn more about the org. I love python and projects around it. Is there any IRC channel like on slack or something.

**Comment by RMeli** [comment]
Hi @octonawish-akcodes, welcome to MDAnalysis!

Please have a look at the [Participating](https://www.mdanalysis.org/#participating) section for a link to our Discord server.

**Comment by octonawish-akcodes** [comment]
Is there any good first issues i can start with?

**Reply by RMeli** [reply]
We have a [good-first-issue label](https://github.com/MDAnalysis/mdanalysis/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue) but good first issues are reasonably scarce lately.

Since you mentioned you are new to the project, I'd strongly recommend to go through our [User Guide](https://userguide.mdanalysis.org/stable/index.html) first, to familiarise yourself with the library and its capabilities and use cases.

The last sections of the User Guide have detailed explanations on [how to contribute to MDAnalysis](https://userguide.mdanalysis.org/stable/contributing.html).
